geode-assembly/build.gradle

```diff
@@ -670,3 +671,8 @@ task dumpInstalledJars(dependsOn: installDist) {
     }
   }
 }
+
+docker {
```

Comment:
The `docker` task should probably depend on whatever task generates the `build/install/apache-geode` directory (maybe `installDist`?) But that could also be a separate PR.

Reply:
Should definitely have a 'dependsOn `installDist`